### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by Maxwell011**
### Hello In this PR I have done the following 👍 

- Set up a React app.
- Set up React linters.
- Followed Git flow.
- set up basic working requirements.

Kindly Review 🙏

### CODE REVIEW

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Maxwell011,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You do not have a descriptive README

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
-# Math-Magician
\ No newline at end of file
+# Getting Started with Create React App
```
Comment (brytebee) [inline README.md:1]: - [ ] **Required:** Kindly update your README with every required section of the [Microverse README template](https://github.com/microverseinc/readme-template) as pointed out by the previous reviewer.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Maxwell011👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — COMMENTED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- 

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**